docs/pages/setup/configuration.mdx

```diff
@@ -0,0 +1,110 @@
+---
+title: Teleport Configuration
+description: A general guide to common Teleport configuration scenarios and questions.
```

Comment:
I think it's a great idea. 

Let's rename this to FAQ, move below the guides and structure as a question/answer format, so people can search and reference it